trackers/javascript-tracker/tracker.config.ts

```diff
@@ -45,4 +45,5 @@ export const plugins = {
   siteTracking: true,
   optimizely: false,
   browserFeatures: false,
+  mediaTracking: true,
```

Comment:
This should be false. We don't want to bundle this plugin into the main `sp.js` for filesize reasons. We'll always suggest it is loaded as a plugin.